doc/man5/config.pod

```diff
@@ -326,69 +330,41 @@ key types, as shown here:
  RSA.Certificate = server-rsa.pem
  ECDSA.Certificate = server-ecdsa.pem
 
-=head2 Engine Configuration
+=head2 Random Configuration
 
-The name B<engines> in the initialization section names the section
-containing the list of ENGINE configurations.
-As with the providers, each name in this section identifies an engine
-with the configuration for that engine.
-The engine-specific section is used to specify how to load the engine,
-activate it, and set other parameters.
+The name B<random> in the initialization section names the section
+containing the random number generater settings.
 
-Within an engine section, the following names have meaning:
+Within the random section, the following names have meaning:
 
 =over 4
 
-=item B<engine_id>
+=item B<random>
 
-This is used to specify an alternate name, overriding the default name
-specified in the list of engines. If present, it must be first.
+This is used to specify the random bit generator.
```

Comment:
Could we list the internally supported DRBG names?

Reply:
I've added a list.  I also included DRBG names in the _cipher_ and _digest_ names.